frontend/src/pages/product-overview/ProductOverview.jsx

```diff
@@ -38,6 +47,28 @@ function ProductOverview() {
       ),
     },
   ];
+
+  const placeBid = async () => {
+    if (enteredPrice < product.startingPrice) {
+      setMessage(
+        "There are higher bids than yours. You could give a second try."
+      );
+      setMessageStyle("error");
+    }
```

Comment:
No need for this frontend check, try to bid and if it fails, display the message you receive from backend.